seed/csharp-sdk/package-yml/src/SeedPackageYml/SeedPackageYmlClient.cs

```diff
@@ -21,7 +21,7 @@ public SeedPackageYmlClient(ClientOptions clientOptions = null)
 
     public ServiceClient Service { get; }
 
-    public async Task<string> EchoAsync(string id, string request)
+    public async Task<string> EchoAsync(string id, EchoRequest request)
```

Comment:
here's the changed test to cover this, apparently it wasn't covered before (having request objects in base client methods)